tck/api/pom.xml

```diff
@@ -23,15 +23,6 @@
 
     <build>
         <plugins>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-compiler-plugin</artifactId>
-                <version>2.3.2</version>
-                <configuration>
-                    <source>1.8</source>
-                    <target>1.8</target>
-                </configuration>
-            </plugin>
```

Comment:
The whole build block can be deleted. You need to rename the overall spec adoc using the naming convention: microrpofile-metrics-spec.asciidoc